internals/detect_intent.py

```diff
@@ -284,3 +286,21 @@ def create_approvals(self, feature: FeatureEntry,
         # REVIEW_REQUESTED votes.
         approval_defs.set_vote(feature_id, approval_field.field_id,
             Vote.REVIEW_REQUESTED, from_addr)
+
+  def record_slo(self, feature, approval_field, from_addr) -> None:
```

Comment:
I remember that we had a script to backfill all the slo by filtering through earliest activities. Should this consider as a part of activities as well?

Reply:
It wouldn't be so easy because it would need to access the Google Groups message archives.  And, I don't think that it would be needed because we don't really need the SLO dates to be accurate on old gates that have been approved already.